the-blue-alliance-ios/View Controllers/Districts/Team@District/DistrictBreakdownViewController.swift

```diff
@@ -183,9 +182,8 @@ extension DistrictBreakdownViewController: Refreshable {
             }
 
             self.persistentContainer.performBackgroundTask({ (backgroundContext) in
-                let backgroundTeam = backgroundContext.object(with: self.ranking.team!.objectID) as! Team
                 if let modelEvent = modelEvent {
-                    backgroundTeam.addToEvents(Event.insert(with: modelEvent, in: backgroundContext))
```

Comment:
Removing this is going to be problematic - the Breakdown VC won't pickup newly added events